.idea/inspectionProfiles/Artemis.xml

```diff
@@ -1504,5 +1505,17 @@
     <inspection_tool class="YAMLSchemaValidation" enabled="false" level="WARNING" enabled_by_default="false" />
     <inspection_tool class="YAMLUnresolvedAlias" enabled="false" level="ERROR" enabled_by_default="false" />
     <inspection_tool class="YAMLUnusedAnchor" enabled="false" level="WARNING" enabled_by_default="false" />
+    <inspection_tool class="unused" enabled="true" level="WEAK WARNING" enabled_by_default="true" editorAttributes="INFO_ATTRIBUTES" checkParameterExcludingHierarchy="false">
```

Comment:
I don't understand what this does..?

Reply:
Warning. : code fragments that might produce bugs or require enhancement. Weak Warning. : code fragments that can be improved or optimized (redundant code, duplicated code fragments, and so on).
https://www.jetbrains.com/help/idea/configuring-inspection-severities.html

Reply:
the unused inspection was level="WARNING" before (default)

Reply:
changed it back to warning but only if the member can't be accessed.